src/Microsoft.Health.Dicom.Core/Modules/ServiceModule.cs

```diff
@@ -139,9 +148,32 @@ public void Load(IServiceCollection services)
                 .AsImplementedInterfaces();
 
             services.Add<QueryTagService>()
-                   .Scoped()
-                   .AsSelf()
-                   .AsImplementedInterfaces();
+                .Scoped()
+                .AsSelf()
+                .AsImplementedInterfaces();
+
+            OperationsConfiguration operationsConfig = _servicesConfiguration.Operations;
+            IEnumerable<TimeSpan> delays = Backoff.ExponentialBackoff(
+                TimeSpan.FromMilliseconds(operationsConfig.MinRetryDelayMilliseconds),
+                operationsConfig.MaxRetries);
+
+            services.AddHttpClient<DicomDurableFunctionsHttpClient>()
+                .AddTransientHttpErrorPolicy(p => p.WaitAndRetryAsync(delays));
+
+            services.Add<DicomDurableFunctionsHttpClient>()
```

Comment:
The `DicomDurableFunctionsHttpClient` will be scoped to [a single request](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0#overview-of-dependency-injection) so that will refreshed often enough to leverage the transient `HttpClient` for connection pooling.